advisories/github-reviewed/2024/01/GHSA-c24v-8rfc-w8vw/GHSA-c24v-8rfc-w8vw.json

```diff
@@ -144,11 +144,9 @@
   ],
   "database_specific": {
     "cwe_ids": [
-      "CWE-178",
```

Comment:
why would you remove these?

https://cwe.mitre.org/data/definitions/178.html
https://cwe.mitre.org/data/definitions/200.html
https://cwe.mitre.org/data/definitions/284.html

all apply to this. Case insensitivity isn't taken into account which gives unauthorized access and might expose sensible information.

Reply:
I am not a bot or a malice. I am Zulu man. Sorry for not replying just tort it wont be necessary but i do apologise.